static/js/forms.js

```diff
@@ -93,34 +93,32 @@
     }
   }
 
-  document.addEventListener('DOMContentLoaded', function(event) {
-    // Do not add search on form pages
-    const search = document.querySelector('.navSearchWrapper.reactNavSearchWrapper');
-    if (search && document.forms.length > 0) {
-      // The following does not work on IE 11: search.remove();
-      search.outerHTML = "";
-    }
+  // Do not add search on form pages
```

Comment:
What I did here was the basics so that the form will redirect, but I still need to connect segment properly so that things are logged.